server/src/handler/mod.rs

```diff
@@ -16,6 +19,18 @@ impl<Deps: TpmContextDeps> CommandHandler<Deps> {
     pub fn new(crypto: Deps::Crypto) -> Self {
         Self { crypto }
     }
+    /// invariant: buffer will fit into TPM2B_DIGEST which is much less
+    /// than the maximum request of SP 800 90A
+    fn infallable_get_random(&mut self, buffer: &mut [u8]) {
+        if self.crypto.fill_bytes(&[], buffer).is_err() {
```

Comment:
Personally, I'd add a comment here to explain this in a little bit more detail, specifically it'd be nice to know how this can happen (the need to reseed). I see *what* we're doing, but not exactly how we got here (or could have gotten here).